src/Smalot/PdfParser/Font.php

```diff
@@ -107,9 +107,15 @@ public function translateChar($char, $use_default = true)
             \strlen($char) < 2
             && $this->has('Encoding')
             && $this->get('Encoding') instanceof Encoding
-            && WinAnsiEncoding::class === $this->get('Encoding')->__toString()
         ) {
-            $fallbackDecoded = self::uchr($dec);
+            try {
+                if (WinAnsiEncoding::class === $this->get('Encoding')->__toString()) {
+                    $fallbackDecoded = self::uchr($dec);
+                }
+            } catch (\Exception $e) {
+                // Encoding->getEncodingClass() throws an exception when BaseEncoding doesn't exists
+                // See table 5.11 on PDF 1.5 specs for more info
+            }
```

Comment:
I get what you want to achieve here but a "catch-all" at this point might hide other viable exceptions. 

What do you think about adapting `getEncodingClass` to throw a specific exception like `EncodingClassNotFoundException`. We could catch that and ignore the rest at this point.

Reply:
I agree, a "catch-all" can create a new bug if any other Exception was raised.